Common/Mpu6050.hpp

```diff
@@ -0,0 +1,196 @@
+#pragma once
+
+#include <cstdint>
+
+#include "L1_Peripheral/i2c.hpp"
+#include "L2_HAL/sensors/movement/accelerometer.hpp"
+#include "utility/bit.hpp"
+#include "utility/enum.hpp"
+#include "utility/map.hpp"
+
+namespace sjsu
+{
+/// Driver for the Mpu6050 3-axis accelerometer
+class Mpu6050 : public Accelerometer
+{
+ public:
+  /// Map of all of the used device addresses in this driver.
+  enum class RegisterMap : uint8_t
+  {
+    /// Register address of the the first byte of the X axis
+    kXYZStartAddress = 0x3B,
+
+    /// Device ID register address
+    kWhoAmI = 0x75,
+
+    /// Device configuration starting address
+    kDataConfig = 0x1C,
+
+  };
+
+  ///
+  ///
```

Comment:
Say something here like

```suggestion
  /// MPU6050 Constructor
  ///
```